finality-aleph/src/network/clique/protocols/v0/mod.rs

```diff
@@ -263,17 +253,18 @@ mod tests {
 
     #[tokio::test]
     async fn parent_service_dead() {
-        let (
-            _id_incoming,
-            _pen_incoming,
-            _id_outgoing,
-            _pen_outgoing,
+        let MockPrelims {
+            id_incoming: _id_incoming,
```

Comment:
The whole point with the `..` is that you can avoid writing all the unused variables.

Reply:
I left them because of the `_`/`_var` distinction, where `_` desugars to `_` and this results in a bunch of failures in the test module.

Reply:
You should  only leave the ones that result in failures – I'm slightly surprized any do, but only the channels are even possible, so probably leave them?

Reply:
OK, so I have now removed all the unused non-channel variables and it works as expected. Removing a random channel variable still results in errors, so unless we want to check all of them in sequence, this seems like a good place to stop :D